test/functional/cmdline_options_tester/src/TestConfigParser.java

```diff
@@ -156,6 +159,10 @@ public void xmlStartElement(String elementName, Hashtable attributes) {
 					String modeHints = (String)attributes.get("modeHints");
 					if (hasAllowedHints(modeHints)) {
 						String timeout = (String)attributes.get("timeout");
+						
+						/* Set 10 days (864000 secs) for timeout on RISC-V due to the lack of JIT */
```

Comment:
Seems a little excessive for a single test to run for 10 days.

Reply:
It won't be necessary in most of cases. The idea is to disable the timer to some extent to ensure we cover all test case in `/test/functional/cmdLineTests`. (e.g. DDR related test which runs extremely slow on the hardware and even more slower on the emulator).

Reply:
Ok, but the timeout is for an individual command line test, not a suite of tests. How long does the DDR test take?

Reply:
Yes, that's we expect to do for individual tests.  According to my previous experience, it took from a couple of hours to half of a day to finish, depending on the size of the generated core dumps.

Reply:
Then you can use something like 16 hours for the timeout, rather than 10 days.

Reply:
Updated to 16 hours as suggested. We still have a chance to adjust in the future if any issue with timeout is detected in there.